src/utils/scratchOrgUtils.ts

```diff
@@ -28,7 +28,7 @@ export default class ScratchOrgUtils {
                                 this.sfdxAuthUrlFieldExists = true;
                             }
 
-                            if (field.name === 'Allocation_status__c' && field.picklistValues.length === 4) {
+                            if (field.name === 'Allocation_status__c' && field.picklistValues.length === 5) {
```

Comment:
This will break backward compatibility, Probably check greater than 4?

Reply:
Or may be as this functionality is not required for sfpowerkit, the ideal fix would be is to work on the warning